styles/main.scss

```diff
@@ -1408,3 +1403,121 @@ blockquote p:last-child {
   overflow: auto;
   max-width: calc(100vw - 30px);
 }
+
+.triangle {
+  width: 0px;
+  height: 0px;
+  position: absolute;
+  z-index: 3;
+}
+
+.triangle-item {
+  border-top: 62px solid transparent;
+  border-bottom: 62px solid transparent;
+}
+
+.triangle-text {
+  display: block;
+  position: relative;
+  transform: rotate(88deg);
+  font-weight: 500;
+  color: white;
+  text-transform: uppercase;
+}
+
+.small-triangles-item {
+  border-top: 5px solid transparent;
+  border-bottom: 5px solid transparent;
+}
+
+.triangle-draft {
+  top: -45px;
+  transform: rotate(225deg);
+  left: -13px;
+  border-left: 62px solid #3a3a3a;
+  z-index: 3;
+}
+
+.triangle-text-draft {
+  right: 42px;
+  font-size: 16px;
+  top: -35px;
+}
+
+.small-triangles-item-draft {
+  border-left: 5px solid rgb(0, 0, 0);
+  transform: rotate(45deg);
+}
+
+.small-triangles-item-draft-first {
+  top: -5px;
+  left: 80px;
+}
+
+.small-triangles-item-draft-second {
+  top: 76px;
+  left: -1px;
+}
+
+.triangle-future {
+  top: -45px;
+  transform: rotate(315deg);
+  right: -13px;
+  border-left: 62px solid #fe6c0a;
+}
+
+.triangle-text-future {
+  right: 37px;
+  font-size: 14px;
+  top: -34px;
+}
+
+.small-triangles-item-future {
+  border-left: 5px solid #a24100;
+  transform: rotate(-225deg);
+}
+
+.small-triangles-item-future-first {
+  top: -5px;
+  right: 79px;
+}
+
+.small-triangles-item-future-second {
+  top: 76px;
+  right: -2px;
+}
+
+.triangle-text-future-post {
+  right: 50px;
+  top: -43px;
+}
+
+.posts-list-block-draft-or-future .post-img:after {
+  border: none;
+  height: 50%;
+}
+
+.posts-list-block-draft-or-future:hover .post-img:after {
+  height: 75%;
+}
+
+.draft-img {
+  background-color: rgba(242, 242, 242, 1);
+  width: 100%;
+  height: 100%;
+  border-radius: 4px;
+}
+
+.draft-img-text {
+  position: relative;
+  top: 50%;
+  color: black !important;
+  text-transform: uppercase;
+  font-weight: 500;
+  font-size: 25px;
+  text-align: center;
+}
+
+.gray-filter-for-img {
+  filter: grayscale(50%);
```

Comment:
![image](https://user-images.githubusercontent.com/46556479/197394126-9afa6679-d57e-49b1-bcaa-bcb36be4e46f.png)
Why you don't use this class here?

It creates code duplication